### PR TITLE
Create board-responsibilities.md

### PR DESCRIPTION
**Pull request by BillWagner**
See https://github.com/dotnet-foundation/wg-membership/pull/63/files#r907743839

Add the existing text for board responsibilities.

@wordshaker I added this file to the same folder for board-recruiting in #63

### CODE REVIEW

File: resources/recruiting-board/board-responsibilities.md
Comment (wordshaker) [inline resources/recruiting-board/board-responsibilities.md:60]: Does it need the time commitment outlined like in the Marketing Committee Chair section?

And should all these sections have the same format

- Overview 
- Main goals in bullet points
- Time Commitment

**Review by wordshaker — CHANGES_REQUESTED** [review]
Really like the content so far. I've asked for some clarifications but they are optional and I'm happy to approve if you don't think they are relevant

File: resources/recruiting-board/board-responsibilities.md
Comment (BillWagner) [reply resources/recruiting-board/board-responsibilities.md:22]: I updated based on current practices. The last bullet is hard to answer because (I hope) the last year was anything but normal.

**Comment by BillWagner** [comment]
@wordshaker I've addressed most of the comments. I think we can make this live and continue to iterate.

File: resources/recruiting-board/board-responsibilities.md
Comment (AraHaan) [reply resources/recruiting-board/board-responsibilities.md:57]: Also can anyone be a part of it as well? Is it paid or unpaid? Are there specific times where we would have to meet if I were to be a part of it? All questions I have for this, I have college so I would need to know if it would conflict with my schedule.

Also do I have to be a member first (currently I am a member but in external CI access so I feel like that is not full membership, just partial)?

Is part of the outreach committee's goals also to "ensure that features that a lot of people would like are part of .NET" so that they would have more motivation to move to .NET as well?

**Review by wordshaker — APPROVED** [review]

**Comment by AraHaan** [comment]
I left a comment about a few questions I have.

File: resources/recruiting-board/board-responsibilities.md
Comment (BillWagner) [reply resources/recruiting-board/board-responsibilities.md:57]: Hi @AraHaan 

These are all good questions. I'll create a new issue to address them. (We'll be updating all the committee info in the next month or so).


> Also can anyone be a part of it as well?
Yes.
> Is it paid or unpaid? 
Unpaid. All the work done by the foundation is unpaid.

> Are there specific times where we would have to meet if I were to be a part of it? 
Each committee sets its meeting schedule, taking into account timezones and availability of the committee members.

> What about traveling? 

Not necessary. All the committees meet over teams.

> Also do I have to be a member first (currently I am a member but in external CI access so I feel like that is not full membership, just partial)?

That's full membership. There's a new membership portal that will be launching soon. That may be what you've heard about.

> Is part of the outreach committee's goals also to "ensure that features that a lot of people would like are part of .NET" so that they would have more motivation to move to .NET as well?

No. The .NET Foundation doesn't set the roadmap for any member projects. That's all done by each projects' maintainers.

**Pull request merged**
